extensions-core/mysql-metadata-storage/src/main/java/org/apache/druid/metadata/storage/mysql/MySQLConnector.java

```diff
@@ -57,11 +58,23 @@ public MySQLConnector(
   {
     super(config, dbTables);
 
+    try {
+      Class.forName(MYSQL_JDBC_DRIVER_CLASS_NAME, false, getClass().getClassLoader());
+    }
+    catch (ClassNotFoundException e) {
+      throw new ISE(e, "Could not find %s on the classpath. The MySQL Connector library is not included in the Druid "
+                   + "distribution but is required to use MySQL. Please download a compatible library (for example "
+                   + "'mysql-connector-java-5.1.38.jar') and place it under 'extensions/mysql-metadata-storage/'. See "
+                   + "http://druid.io/downloads for more details.",
```

Comment:
https://druid.apache.org/downloads (under the assumption we'll get this live when we actually release this)?